app/Helpers/MiscHelper.php

```diff
@@ -2,20 +2,20 @@
 
 namespace App\Helpers;
 
-class MiscHelper{
+class MiscHelper {
 
-    public static function array_diff_assoc_recursive($array1, $array2){
-        $difference=array();
-        foreach($array1 as $key => $value) {
-            if( is_array($value) ) {
-                if( !isset($array2[$key]) || !is_array($array2[$key]) ) {
+    public static function array_diff_assoc_recursive($array1, $array2) {
+        $difference = array();
+        foreach ($array1 as $key => $value) {
+            if (is_array($value)) {
+                if (!isset($array2[$key]) || !is_array($array2[$key])) {
                     $difference[$key] = $value;
-                } else {
+                }else {
                     $new_diff = MiscHelper::array_diff_assoc_recursive($value, $array2[$key]);
-                    if( !empty($new_diff) )
+                    if (!empty($new_diff))
                         $difference[$key] = $new_diff;
                 }
-            } else if( !array_key_exists($key,$array2) || $array2[$key] !== $value ) {
+            }else if (!array_key_exists($key, $array2) || $array2[$key] !== $value) {
```

Comment:
__[code_sniffer]__
* Expected 1 space after closing brace; 0 found
* Usage of ELSE IF is discouraged; use ELSEIF instead

<!-- Comment by [SideCI](https://sideci.com) -->